daemon/mgr/container_utils.go

```diff
@@ -379,3 +392,20 @@ func amendContainerSettings(config *types.ContainerConfig, hostConfig *types.Hos
 		r.MemorySwap = 2 * r.Memory
 	}
 }
+
+// isCFQScheduler checks if dev slice is all cfq io scheduler
+func isCFQScheduler(dev []string) bool {
```

Comment:
I think we need unit test for this function for sure.

Reply:
If dev slice is null, it is reasonable to return true, only a device is not use cfq will return false, other case include error happens, we all return true. We just want make sure for not cfq.
I think this function is not suit for ut, since the way we get which io scheduler device uses will same as the way function uses. Or maybe you can get some good idea to add a ut ?